wordpress-mobile/WordPress-iOS
# Notification Likes list: show user profile in bottom sheet when user tapped.

Ref #16244 

In the user list for a Like Notification, a user profile bottom sheet is shown when the user row is tapped. The view currently only shows:
- The "user information" block - gravatar, name, username, and a stub for the user bio.
- Section header for the site section.

The bottom sheet should render correctly on iPhone in portrait and landscape.

To be addressed in follow-up PRs:
- Correctly placing the sheet on iPad.
- Adding a table row for the user's site (to be displayed in the currently empty site section).

To test:
- Enable the `newLikeNotifications` feature.
- Run on an iPhone. (It will show on an iPad, just not in the right place.)
- Go to Notifications > Likes and tap a notification.
- Tap a user.
  - Verify the bottom sheet appears as described above.
  - Verify the sheet is partial height in portrait.
  - Verify the sheet is full height in landscape.
- Disable the `newLikeNotifications` feature.
- Go to Notifications > Likes and tap a notification.
  - Verify tapping the user opens the user's site.

| ![light_portrait](https://user-images.githubusercontent.com/1816888/114245476-c9acab00-994d-11eb-83b3-1de257b8b769.png) | ![dark_portrait](https://user-images.githubusercontent.com/1816888/114245482-cd403200-994d-11eb-8c83-483bad02ccfe.png) |
|--------|-------|
| ![light_landscape](https://user-images.githubusercontent.com/1816888/114245500-d5986d00-994d-11eb-9de7-5076c13c451f.png) | ![dark_landscape](https://user-images.githubusercontent.com/1816888/114245505-d92bf400-994d-11eb-935c-30566a5928e5.png) |


## Regression Notes
1. Potential unintended areas of impact
N/A. Feature is incomplete.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A. Feature is incomplete.

3. What automated tests I added (or what prevented me from doing so)
N/A. Feature is incomplete.

PR submission checklist:
